newmiddleware/formsg.ts

```diff
@@ -0,0 +1,24 @@
+import autoBind from "auto-bind"
```

Comment:
could i ask why the file is lowercased rather than having it in sentence case as it exports a class

Reply:
Will be removed